command/ssh.go

```diff
@@ -460,10 +470,6 @@ func (c *SSHCommand) handleTypeCA(username, hostname, ip string, sshArgs []strin
 		)
 	}
 
-	args = append(args,
```

Comment:
Why is this removed entirely? We don't need to pass in the username, IP, and port to the SSH command explicitly any more?

Reply:
Correct. My comment here should clear things up for you: https://github.com/hashicorp/vault/pull/4710#issuecomment-395293343

The [current behaviour](https://github.com/Crazybus/vault/blob/2b7440524e2ff9d384a7aa61ff148dec2a018a14/command/ssh.go#L253-L257) was to expect username@hostname as the first argument. This was then added to the actual ssh command with username@hostname plus the remaining ssh arguments. This PR removes that limitation of needing to use username@hostname and instead allows you to use any valid ssh command so that vault doesn't need to modify the actual command.